src/inmanta/moduletool.py

```diff
@@ -507,6 +508,14 @@ def modules_parser_config(cls, parser: ArgumentParser) -> None:
             default=None,
             dest="output_dir",
         )
+        build.add_argument(
+            "--dev",
+            dest="dev_build",
+            help="Perform a development build of the module. This adds the build tag `.dev<timestamp>` to the "
+            "package name. The timestamp has the form %%Y%%m%%d%%H%%M%%S.",
```

Comment:
Out of curiosity, how does this work with the double percent signs? I got the impressino this is only required if you actually use the `%` operator on it. But the help string actually shows the expected output. Does `ArgumentParser` use `%`?
```python
>>> "%%s%s" % "test"
'%stest'
>>> "%stest"
'%stest'
>>> "%%stest"
'%%stest'
```

Reply:
Yes. The `%` character is used by argparse to reference for example the default value of the option in the help string. If you want to use a literal `%` character, it has to be escaped with a double `%` character.